src/ui/components/CollectionFilters/CollectionFilters.tsx

```diff
@@ -109,6 +117,17 @@ export const CollectionFilters = React.memo<Props>(
             [handleChangeFilters],
         );
 
+        const handleChangeView = React.useCallback(
+            (value) => {
+                handleChangeFilters({
+                    viewMode: value,
+                });
+
+                Utils.store('viewMode', value);
```

Comment:
`viewMode` should be `const` and have a more specific name, for example – `collectionPageViewMode`